torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -428,11 +435,12 @@ struct NCCLTraceBuffer {
     size_t pg_id_;
     std::tuple<std::string, std::string> pg_name_; // <group_name, group_desc>
 
-    // Both seq_id_ and op_id_ are per_pg incrementing counters
-    // seq_id refers to actual kernel launches (e.g. 1 per coalesced group)
-    // op_id refers to logical operations (e.g. one per op inside coalesced
-    // group)
-    size_t seq_id_;
+    // Both collective_seq_id_ and op_id_ are per_pg incrementing counters
+    // collective_seq_id refers to actual kernel launches (e.g. 1 per coalesced
```

Comment:
collective_seq_id and p2p_seq_id refer to actual kernel launches (e.g. 1 per coalesced..), however collective_seq_id only increments for true collective operations (over all ranks in the group) while p2p_seq_id increments only over non-collective operations in the group.

Reply:
Fixed the comment. Thanks.